mobile/src/modules/auth/screens/HomeScreen.tsx

```diff
@@ -104,9 +107,24 @@ class Home extends React.PureComponent<TProps, State> {
 
   render () {
     const accounts: Account[] = this.props.auth.accounts || [];
+    const totalBalance = accounts.map(({ balanceNQT }) => balanceNQT)
```

Comment:
Also i think we should use Big.js for very large numbers. Do we have any problems with them in desktop wallet?

Reply:
We do have some issues with very small numbers being represented in e notation in burstJS, but this seems to work fine for now.